lnbits/extensions/watchonly/views_api.py

```diff
@@ -76,7 +73,8 @@ async def api_wallet_create_or_update(
     data: CreateWallet, w: WalletTypeInfo = Depends(require_admin_key)
 ):
     try:
-        (descriptor, network) = parse_key(data.masterpub)
+        # TODO: talk to motorina about this
+        (descriptor, network) = parse_key(data.masterpub)  # type: ignore
```

Comment:
@motorina0 do you know how to fix this one?

Reply:
- `# type: ignore` in this case is redundant for the `mypy` execution
- removed, see commit